.github/workflows/ci.yml

```diff
@@ -44,20 +47,3 @@ jobs:
         run: yarn test:ceramic-integration
         env:
           CERAMIC_CLIENT_URL: http://localhost:7007
-
```

Comment:
I would expect this to not be removed.
If we have a separate action for this, invoke it here

Reply:
So we want to check the provider bitmaps at every PR ?

Reply:
Yes.